doc/development-guide.md

```diff
@@ -104,3 +104,22 @@ ensure the following:
 * End all files with a newline character. See the POSIX definition of a
   [line](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_205).
 * Add a changelog entry, linking to your pull request.
+
+## OpenDevStack branching flow for contributions
+
+Next sequence diagram provides a graphical understanding on the steps to follow in order to add a new functionality to the OpenDevStack official repository from ACME's private fork.
+The example is based on the assumption you want to contribute with a new feature from branch *1.0.x* or tag *v1.0.x*
+
+<div class="mermaid">
+sequenceDiagram
+    participant A as OpenDevStack <br />GitHub
+    participant B as ACME GitHub <br />OpenDevStack fork
+    participant C as ACME private repo <br />OpenDevStack fork
+    A->>C: 1.0.x/v1.0.x checkout
```

Comment:
I think this should be `A->>C: 1.0.x/v1.0.1 checkout` to show a concrete example.

Reply:
done